apps/ff-site/src/content/events/fil-paris.md

```diff
@@ -1,17 +1,327 @@
 ---
-title: FIL Paris 2023
-created-on: 2023-06-27T15:38:07.082Z
-updated-on: 2023-06-27T15:38:07.098Z
-published-on: 2023-06-27T15:38:07.109Z
+title: FIL Paris
+description: FIL Paris 2023 was a week-long event (July 15–22) in Paris,
+  celebrating the Filecoin ecosystem and the decentralized web. Hosted by the
+  Filecoin Foundation and Protocol Labs, it brought together developers,
+  researchers, and Web3 enthusiasts for talks, workshops, and networking.
+created-on: 2025-04-29T09:47:00.000Z
+updated-on: 2025-04-29T09:47:00.000Z
+published-on: 2025-04-29T09:47:00.000Z
 category: hosted
 location:
   primary: Paris, France
   region: europe
 start-date: 2023-07-15
 end-date: 2023-07-22
 image:
-  src: /assets/images/fil-paris-pic.png
+  src: /assets/images/fil-paris-pic-1-.webp
+program:
+  events:
```

Comment:
@CharlyMartin - The cards are showing hover styles even when there's no link applied.